test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -155,6 +153,32 @@ def load_schemas_to_dict():
         new_schema_dict[s.name].append(s)
     return new_schema_dict
 
+def has_valid_upgraders(schema):
+    version_map = torch._C._get_operator_version_map()
+    # we want to parse through the map to find if
+    # the schema has valid upgraders. Since the
+    # version map has entry for each overload
+    # we need to do some ugly parsing.
+
+    # the name of the operator
+    schema_name = str(schema).split('(')[0]
+
+    # find if there are entries for this op
+    possible_overloads = set()
+    for key in version_map:
+        if key.split('.')[0] == schema_name:
+            possible_overloads.add(key)
+
+    # there is no entry for this schema
+    if len(possible_overloads) == 0:
```

Comment:
Do we need to verify that these upgraders actually have same schema (not just same op name) as `existing_schema`?